howso/typing.amlg

```diff
@@ -1149,6 +1149,27 @@
                         "If all, then returns all including the confusion_matrix."
                     )
             )
+        DesiredConviction
+            (assoc
+                type ["number" "null"]
+                min 0.000001
```

Comment:
Might be better to have an `exclusiveMin` and `exclusiveMax` to handle this better. Then you could have `exclusiveMin: 0` for values > 0